boards/arm/adafruit_feather_nrf52840_sense/Kconfig

```diff
@@ -0,0 +1,9 @@
+# Adafruit Feather nRF52840 Sense board configuration
+
```

Comment:
I wasn't sure what to do with the [copyright notices](https://www.linuxfoundation.org/blog/blog/copyright-notices-in-open-source-software-projects), so I mostly deleted them. I'll be happy to add them back if that was incorrect